_includes/cockroachcloud/restore-examples.md

```diff
@@ -1,6 +1,6 @@
 #### View the backup subdirectories
 
-<span class="version-tag">New in v21.1:</span> `BACKUP ... INTO` adds a backup to a collection within the backup destination. The path to the backup is created using a date-based naming scheme. To view the backup paths in a given destination, use [`SHOW BACKUPS`](../{{site.versions["stable"]}}/restore.html#view-the-backup-subdirectories):
+`BACKUP ... INTO` adds a backup to a collection within the backup destination. The path to the backup is created using a date-based naming scheme. To view the backup paths in a given destination, use [`SHOW BACKUPS`](../{{site.versions["stable"]}}/restore.html#view-the-backup-subdirectories):
```

Comment:
perhaps worth clarifying the date based scheme is just the default behavior. Second sentence could be instead:

"The path to the backup is created using a date-based naming scheme by default, unless the user passed an explicit subdirectory during the BACKUP statement"

Reply:
Can we add an example of what that date-based naming scheme looks like for full as well as incremental backups.

Reply:
Hey @livlobo I have added an update for this. Let me know what you think, thanks!

Reply:
Thanks @kathancox - LGTM!